runtime/onert/sample/minimal-python/README.md

```diff
@@ -11,3 +11,57 @@ It assumes model of float32 tensor type as an input.
 ```
 $ python3 minimal.py path_to_nnpackage_directory
 ```
+
+## Classes
```

Comment:
```suggestion
## Class
```

There is only 1 class.

Reply:
`nnfw_session` seems to be a part of NNFW Python API. Why `README.md` for `minima-python` needs to have this?

Reply:
I think `nnfw_session` needs a guide because inheritance of `nnfw_session` which bound in c++ makes differences from traditional `nnfw capi`.

Reply:
I am saying why it is located in runtime/onert/**sample**/minimal-python/README.md.

Also, don't resolve conversation, it is not resolved at all.

Reply:
I'm so sorry. It is automatically resolving conversation when I commit suggestion.

And I thought this file only uses `nnfw python api`, so needs a guide about this differences.
If you think this is not requirable, I'll remove this or locate other directory if you want.

Reply:
The documentation for API itself needs to be located out of `sample`. Also, reference to the API document may be helpful, but I don't want to make duplication. It is maintenance burden.

Reply:
I understand. Then I will remove the description of API in `sample`.